.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.0.4
```

Comment:
I think we probably don't want to commit this, unless we wanted to provide a different way of getting the app set up for development. At the very least, we probably want to make sure that we specify a consistent version of Ruby across all the different environments, and it looks like Dockerfile.dev uses Ruby 2.7.6.

Reply:
Ah, whoops. This was committed accidentally